CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,25 @@
+#Code of Conduct
+
+## Summary
+
+Harassment in code and discussion or violation of physical boundaries is completely unacceptable anywhere in Opal’s project codebases, issue trackers, chatrooms, mailing lists, meetups, etc. Violations will result in being warned, blocked or banned by the core team at or before the 3rd violation.
+
+## In detail
+
+Harassment includes offensive verbal comments related to gender identity, gender expression, sexual orientation, disability, physical appearance, body size, race, religion, sexual images, deliberate intimidation, stalking, sustained disruption, and unwelcome sexual attention.
+
+Individuals asked to stop any harassing behavior are expected to comply immediately.
+
+Maintainers are also subject to the anti-harassment policy.
+
+If anyone engages in harassing behavior, including maintainers, we may take appropriate action, up to and including warning the offender, deletion of comments, removal from the project’s codebase and communication systems, and escalation to GitHub support.
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of the core team or email <conduct@opalrb.org> immediately.
+
+We expect everyone to follow these rules in any and all public spaces associated with the Opal project, including codebases, issue trackers, chatrooms, social media, and mailing lists. 
```

Comment:
Since this covers the scope, I suggest we move it up to the first item in "In Detail".